content/security/docs/iam.md

```diff
@@ -462,18 +464,121 @@ AWS_WEB_IDENTITY_TOKEN_FILE=/var/run/secrets/eks.amazonaws.com/serviceaccount/to
+
+To learn more about EKS Pod Identities, please see [this blog](https://aws.amazon.com/blogs/containers/amazon-eks-pod-identity-a-new-way-for-applications-on-eks-to-obtain-iam-credentials/).
+
+You do not have to make any modifications to your application code to use EKS Pod Identities. Supported AWS SDK versions will automatically discover credentials made available with EKS Pod Identities by using the [credential provider chain](https://docs.aws.amazon.com/sdkref/latest/guide/standardized-credentials.html). Like IRSA, EKS pod identities sets variables within your pods to direct them how to find AWS credentials.
+
+#### Working with IAM roles for EKS Pod Identities
+
+* EKS Pod Identities can only directly assume an IAM role that belongs to the same AWS account as the EKS cluster. To access an IAM role in another AWS account, you must assume that role by [configuring a profile in your SDK configuration](https://docs.aws.amazon.com/sdkref/latest/guide/feature-assume-role-credentials.html), or in your [application's code](https://docs.aws.amazon.com/IAM/latest/UserGuide/sts_example_sts_AssumeRole_section.html).
+* When EKS Pod Identities are being configured for Service Accounts, the person or process configuring the Pod Identity Association must have the `iam:PassRole` entitlement for that role.
+* Each Service Account may only have one IAM role associated with it through EKS Pod Identities, however you can associate the same IAM role with multiple service accounts.
+* IAM roles used with EKS Pod Identities must allow the `pods.eks.amazonaws.com` Service Principal to assume them, *and* set session tags. The following is an example role trust policy which allows EKS Pod Identities to use an IAM role:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "pods.eks.amazonaws.com"
+      },
+      "Action": [
+        "sts:AssumeRole",
+        "sts:TagSession"
+      ],
+      "Condition": {
+        "StringEquals": {
+          "aws:SourceOrgId": "${aws:ResourceOrgId}"
+        }
+      }
+    }
+  ]
+}
+```
+AWS recommends using condition keys like `aws:SourceOrgId` to help protect against the [cross-service confused deputy problem](https://docs.aws.amazon.com/IAM/latest/UserGuide/confused-deputy.html#cross-service-confused-deputy-prevention). In the above example role trust policy, the `ResourceOrgId` is a variable equal to the AWS Organizations Organization ID of the AWS Organization that the AWS account belongs to. EKS will pass in a value for `aws:SourceOrgId` equal to that when assuming a role with EKS Pod Identities.
```

Comment:
@liwadman Is using this condition in the trust policy needed at all?

It's not mentioned in https://docs.aws.amazon.com/eks/latest/userguide/pod-id-association.html and from my understanding it's not adding any extra protections at all.

cc @micahhausler

Reply:
SourceAccount/SourceOrgId/SourceOrgPaths are set whenever an AWS service principal interacts with a customer resource (role, bucket, etc) unless otherwise documented. Our official guidance is to always include them to hlep protect against cross-service confused deputy issues.

Reply:
I will be opening a ticket with the documentation team to add a sample with one of the source condition keys.

Reply:
How about:
```diff
- AWS recommends using condition keys ...
+ AWS recommends but does not require using condition keys like ...
```

also could mention SourceArn equality as an option if only used for one cluster
```json
{
  "ArnEquals": {
    "aws:SourceArn": [
      "arn:aws:eks:REGION:ACCOUNT_ID:cluster/CLUSTER_NAME",
      "arn:aws:eks:REGION:ACCOUNT_ID:cluster/CLUSTER_NAME_2"
    ]
  }
}
```